### DISCUSSION TITLE
General comments post meeting

### DISCUSSION THREAD

**Discussion by taslandcover** [christopherowers/supratidal_forests#7] [General]
Thanks Chris/Rafael/Jeff for the overview of the Supratidal Forests project. I thought we should provide some comments, in case they prove useful!  

- We appreciate that the conceptual model for delineating supratidal forests is necessarily quite simple. We do feel that some elevation cutoffs could lead to the inclusion of unintended ecosystems. The 10m threshold would certainly include many local (Tasmanian) communities that do not fit the description of supratidal forest. Our vegetation scientists note that many local wetlands transition quite abruptly to dry eucalypt forest and a lot of forest would be captured below 10m. It would complicate workflows, but local DEMs can provide increased resolution to better define low-lying coastal features (certainly this is true for TAS).
- I understand the current woody extent layer allows flexibility in setting cover thresholds for 'forest/non forest' but is it worth considering the '[National Forest and Sparse Woody Vegetation](https://researchdata.edu.au/national-forest-sparse-2020-release/1733697)' layer for this purpose? This layer is already used for National Inventory Reports (greenhouse gas accounting) and could facilitate alignment/consistency with broader accounting frameworks.
- TAS certainly has some mapping information that could be useful. Our TASVEG layer identifies vegetation communities (though only a few would be considered as true supratidal forest). This information is used to create NVIS so (in theory) these data should align. Much of the saltmarsh mapping has been recently updated so that could prove useful as exclusion features.

We have limited resourcing capability, but would be happy to provide comments on draft products if necessary : )

**Comment by Rafcarv** [comment]
Hi taslandcover,
Many thanks for your post and kicking off the STF discussions in Github.
Yes, we are still working on the type of elevation dataset (e.g. SRTM, LiDAR, WCF, GMW) and threshold. This has been the topic of several discussions in the past 3-4 months. I'm with you that the 10 m threshold might not work for some of the coastal settings. We have trialed different ones (e.g. 5 m AHD) and it seems that we have different thresholds for different parts of the country. However, we highly appreciate your insights and suggestions. One of the challenges that we face in this project is to derive a national consistent dataset. We would love to use LiDAR or other sort of higher spatial resolution dataset, but the lack of completeness makes them difficult to use at continental-scale.
Thanks also for suggesting the National Forest and Sparse Woody Vegetation and TASVEG layers. We will definitely use it in the cal/val phase starting in early Jan/2024. We hope some of the state maps can help us to refine our model extent (including thresholds!). We will reach out for further comments in the near future. Cheers, Rafael  ;-)
